src/coreaudiokit.cs

```diff
@@ -240,4 +241,18 @@ interface CAInterAppAudioTransportView {
 		void SetOutputAudioUnit (AudioUnit.AudioUnit audioUnit);
 	}
 #endif
+
+	[Mac (13,0), iOS (16,0)]
+	[BaseType (typeof(UIViewController))]
+	interface AUGenericViewController
+	{
+
+		[DesignatedInitializer]
+		[Export ("initWithNibName:bundle:")]
+		[PostGet ("NibBundle")]
```

Comment:
What's this PostGet for?

Reply:
I'm re-exposing the same constructor from the base, I think we can remove the extra attr.

Reply:
```suggestion
```